ble/DiscoveredCharacteristic.h

```diff
@@ -110,13 +108,13 @@ class DiscoveredCharacteristic {
     /**
      * Initiate a GATT Characteristic Descriptor Discovery procedure for descriptors within this characteristic.
      *
-     * @param  callback
-     * @param  matchingUUID
-     *           Filter for descriptors. Defaults to wildcard which will discover all descriptors.
+     * @param onCharacteristicDiscovered This callback will be called every time a descriptor is discovered
+     * @param onTermination This callback will be called when the discovery process is over.
      *
      * @return  BLE_ERROR_NONE if descriptor discovery is launched successfully; else an appropriate error.
      */
-    ble_error_t discoverDescriptors(DescriptorCallback_t callback, const UUID &matchingUUID = UUID::ShortUUIDBytes_t(BLE_UUID_UNKNOWN)) const;
+    ble_error_t discoverDescriptors(const CharacteristicDescriptorDiscovery::DiscoveryCallback_t& onCharacteristicDiscovered, 
```

Comment:
shouldn't the parameter be named `onDescriptorDiscovered` instead of `onCharacteristicDiscovered`?